api/src/services/spatial-service.ts

```diff
@@ -62,24 +74,38 @@ export class SpatialService extends DBService {
    * Collect transform from db, run transform on submission id, save result on spatial component table
    *
    * @param {number} submissionId
-   * @param {number} spatialTransformId
-   * @return {*}  {Promise<{ submission_spatial_component_id: number }>}
+   * @param {string} spatialTransformName
+   * @return {*}  {Promise<void>}
    * @memberof SpatialService
    */
-  async runTransform(
-    submissionId: number,
-    spatialTransformId: number
-  ): Promise<{ submission_spatial_component_id: number }> {
-    const spatialTransform = await this.getSpatialTransformBySpatialTransformId(spatialTransformId);
+  async runTransform(submissionId: number, spatialTransformName: string): Promise<void> {
```

Comment:
If this is specific to spatial transforms, lets change the name to something like `runSpatialTransform`